packages/guides/src/Nodes/FieldLists/FieldListItemNode.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link https://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides\Nodes\FieldLists;
+
+use phpDocumentor\Guides\Nodes\CompoundNode;
+use phpDocumentor\Guides\Nodes\Node;
+
+/**
+ * @extends CompoundNode<Node>
+ */
+final class FieldListItemNode extends CompoundNode
+{
+    private string $term;
+    private string $plaintextContent;
+
+    /**
+     * @param Node[] $children
```

Comment:
Are all nodes allowed? or just metadata nodes?

Reply:
A field list that is located before the page title is treated as metadata. It does not create a FieldListItemNode. A field-list later in the document is just output in a way defined by the project. Most HTML-based output is using tables here.